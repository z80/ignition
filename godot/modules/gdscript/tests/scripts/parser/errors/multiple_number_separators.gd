func test():
	# Number separators may not be placed right next to each other.
	var __ = 1__23
