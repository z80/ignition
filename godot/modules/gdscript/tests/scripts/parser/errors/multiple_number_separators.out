GDTEST_PARSER_ERROR
Only one underscore can be used as a numeric separator.
